src/assessor_portal.yml

```
@@ -16,10 +16,14 @@
           ansible.builtin.command:
             chdir: "{{ assessor_portal_dir }}"
             cmd: "{{ assessor_portal_dir }}/run.py pull"
+        # The community.docker.docker_compose module exists, but it
+        # only supports docker-compose and not the Docker compose
```

Just lining up with their documentation.

```suggestion
        # only supports docker-compose and not the Docker Compose
```